src/assets/scripts/client/constants/gameOptionConstants.js

```diff
@@ -34,6 +35,16 @@ export const GAME_OPTION_VALUES = [
         ]
     },
     {
+        name: GAME_OPTION_NAMES.PTL_LENGTH,
+        defaultValue: '1',
+        description: 'Projected Track Line (PTL) Length',
+        type: 'select',
+        data: [
+            ['1', '1'],
+            ['0.5', '0.5']
```

Comment:
yeah, reworking will let me use option displays like `30sec` and `1min` but use values of `0.5` and `1` respectively.  This also means I can get rid of the `_startCase()` stuff in there.